.github/workflows/rebuild.yaml

```diff
@@ -0,0 +1,82 @@
+---
+
+name: Rebuild
+
+on:
+  schedule:
+    - cron: "30 2 * * *"
+
+jobs:
+  rebuild:
+    runs-on: ubuntu-18.04
+    name: Rebuild
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        branch:
+          - 1.12
+          - 1.13
+
+    env:
+      REDIS_URL: redis://localhost:6379
+      PATH: /bin:/usr/bin:/usr/local/bin:/home/runner/.local/bin
+      SUMMON_PROVIDER: /usr/local/bin/gopass
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          ref: ${{ matrix.branch }}
+
+      - run: sudo apt update
+      - run: sudo apt install python3-wheel python3-setuptools
+
+      - run: python3 -m pip install --requirement=dev-requirements.txt
+
+      - run: touch tilecloud_chain/OpenLayers.js
+
+      - name: Build
+        run: |
+          flake8 tilecloud_chain
+          vvv-validate-rst README.rst
+          vvv-validate-rst CHANGES.rst
+          python setup.py --long-description > PYPI.rst
+          vvv-validate-rst PYPI.rst
+          docker build --tag camptocamp/tilecloud-chain:latest .
+          git --no-pager diff --check `git log --oneline | tail -1 | cut --fields=1 --delimiter=' '`
+
+      - name: Test
+        run: |
+          docker-compose --project-name=tilecloud --file=test-docker-compose.yaml up -d
+
+          # Wait for DB to be up
+          while ! docker exec tilecloud_test_1 psql -h db -p 5432 -U postgres -v ON_ERROR_STOP=1 -c "SELECT 1" -d tests
+          do
+            echo "Waiting for DB to be UP"
+            sleep 1
+          done
+
+          docker-compose --project-name=tilecloud --file=test-docker-compose.yaml exec -T test python setup.py nosetests --logging-filter=tilecloud,tilecloud_chain --attr '!'nopy3
+
+          docker-compose --project-name=tilecloud --file=test-docker-compose.yaml down
+
+      - uses: camptocamp/initialise-gopass-summon-action@v1
+        with:
+          ci-gpg-private-key: ${{secrets.CI_GPG_PRIVATE_KEY}}
+          github-gopass-ci-token: ${{secrets.GITHUB_GOPASS_CI_TOKEN}}
+        if: github.repository == 'camptocamp/tilecloud-chain'
+      - run: |
+          summon --yaml '
+              USERNAME: !var gs/ci/pypi/username
+              PASSWORD: !var gs/ci/pypi/password
+            ' bash -c 'envsubst < ci/pypirc.tmpl > ~/.pypirc'
+        if: github.repository == 'camptocamp/tilecloud-chain'
+      - run: |
+          mkdir  ~/.docker
+          summon --yaml '
+              TOKEN: !var gs/ci/dockerhub/token
+            ' bash -c 'envsubst < ci/docker-config.json.tmpl > ~/.docker/config.json'
+        if: github.repository == 'camptocamp/tilecloud-chain'
+
+      - name: Publish
+        run: ci/publish
```

Comment:
How do you pass the branch (i.e. tag) to publish?

Reply:
Thanks, done :-)